library-app/src/App.tsx

```diff
@@ -1,15 +1,10 @@
-import Homepage from './components/pages/Homepage/Homepage'
 import styles from './App.module.css'
-import { Route, Routes } from 'react-router-dom'
-import Login from './components/pages/Login/Login'
+import MainLayout from './components/Layout/MainLayout'
 
 function App() {
   return (
     <div className={styles['app-wrapper']}>
-      <Routes>
-        <Route path='/' element={<Homepage />} />
-        <Route path='login' element={<Login />} />
```

Comment:
Where did login go? If main layout is layout with header and footer (or sidebar) and login has none of this, why is it moved to main layout?

Reply:
login has both, header on desktop and header and footer on mobile version